bind-mount.c

```diff
@@ -376,6 +375,7 @@ parse_mountinfo (int  proc_fd,
 
 bind_mount_result
 bind_mount (int           proc_fd,
+            int           p_priv,
```

Comment:
This seems like it should be a `BIND_PRIVATE` flag in `bind_option_t options`. It's a lot clearer what the flag means if you see

```
bind_mount (..., BIND_PRIVATE);
```

rather than

```
bind_mount (..., 1, ...);    /* what does 1 mean? is it a boolean? is it a length? ... */
```

or

```
bind_mount (..., TRUE, ...);    /* what is being set to true here? */
```